### PR TITLE
feat: add ability to post and update github status to our repos that …

### PR DESCRIPTION
**Pull request by jhillacre**
…are not the running repo.

### CODE REVIEW

**Comment by jhillacre** [comment]
published as `arrai/github@dev:1`. don't have a way to use it yet.

**Comment by jhillacre** [comment]
updated to `arrai/github@dev:2`

**Comment by jhillacre** [comment]
published as `arrai/github@2.1.0`

File: orbs/github.yml [hunk 1]
```diff
@@ -3,6 +3,124 @@ executors:
...
         docker:
             - image: cimg/base:current
+commands:
+    post-status:
+        description: post status to github
+        # samsalisbury/github-status - github-status/post-status
+        # customized to have a configurable repo and commit
+        parameters:
+            when:
+                type: string
```
Comment (pmack) [inline orbs/github.yml:13]: Should be `type` `enum`?

File: orbs/github.yml [hunk 2]
```diff
@@ -3,6 +3,124 @@ executors:
...
+                enum:
+                    - always
+                    - on_success
+                    - on_fail
+            state:
+                type: string
```
Comment (pmack) [inline orbs/github.yml:19]: This also ought to be `enum`

**Review by pmack — CHANGES_REQUESTED** [review]
I guess change those `string` to `enum`. Not sure why their validator accepts this...

File: orbs/github.yml (hunk 1 above)
Comment (jhillacre) [reply orbs/github.yml:13]:
```suggestion
                type: enum
```

File: orbs/github.yml (hunk 2 above)
Comment (jhillacre) [reply orbs/github.yml:19]:
```suggestion
                type: enum
```

**Comment by jhillacre** [comment]
`arrai/github@2.1.1`

**Review by pmack — APPROVED** [review]
LGTM

**Pull request merged**